nimterop/cimport.nim

```diff
@@ -74,6 +72,7 @@ proc getCacheValue(fullpath: string): string =
     result = fullpath.getFileDate()
 
 proc getToastError(output: string): string =
+  echo "getToastError:\n" & output & "\nfiltered output:\n"
   # Filter out preprocessor errors
```

Comment:
Will cause output to come out twice. Do we really need full preprocessor output?

Reply:
it helped me debug what was going on when I had a failure, but fine, removed

Reply:
EDIT: because of this: when appveyor fails I have no idea why:

https://ci.appveyor.com/project/genotrance/nimterop/builds/21961530/job/ob0g80yxkkha6m9k



```
Hint: colors [Processing]
Importing c:/binaries/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.1.0/../../../../i686-w64-mingw32/include/math.h
(Field0: "getToastExe", Field1: "nim c -o:C:\\Users\\appveyor\\.nimble\\pkgs\\nimterop-0.1.0\\nimterop\\nimterop\\toast C:\\Users\\appveyor\\.nimble\\pkgs\\nimterop-0.1.0\\nimterop\\toast.nim")
cmd /c C:\Users\appveyor\.nimble\pkgs\nimterop-0.1.0\nimterop\nimterop\toast --pnim --preprocess --pluginSourcePath=C:\Users\appveyor\AppData\Local\Temp\1\nimterop_3688256538.nim c:/binaries/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.1.0/../../../../i686-w64-mingw32/include/math.h
stack trace: (most recent call last)
..\..\..\Users\appveyor\.nimble\pkgs\nimterop-0.1.0\nimterop\cimport.nim(448) cImport
..\..\..\Users\appveyor\.nimble\pkgs\nimterop-0.1.0\nimterop\cimport.nim(132) getToast
..\..\..\..\c:\binaries\nim-0.19.2\lib\system.nim(3877) failedAssertImpl
..\..\..\..\c:\binaries\nim-0.19.2\lib\system.nim(3870) raiseAssert
..\..\..\..\c:\binaries\nim-0.19.2\lib\system.nim(2916) sysFatal
tmath.nim(20, 9) template/generic instantiation from here
c:\binaries\nim-0.19.2\lib\system.nim(2916, 7) Error: unhandled exception: C:\Users\appveyor\.nimble\pkgs\nimterop-0.1.0\nimterop\cimport.nim(132, 12) `ret == 0` # Generated at 2019-01-29T08:44:43+00:00
# Command line:
#   C:\Users\appveyor\.nimble\pkgs\nimterop-0.1.0\nimterop\nimterop\toast.exe --pnim --preprocess --pluginSourcePath=C:\Users\appveyor\AppData\Local\Temp\1\nimterop_3688256538.nim c:/binaries/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.1.0/../../../../i686-w64-mingw32/include/math.h
{.experimental: "codeReordering".}
No stack traceback available
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
stack trace: (most recent call last)
C:\projects\nimterop\nimterop.nimble(54) testTask
C:\projects\nimterop\nimterop.nimble(41) testAll
C:\projects\nimterop\nimterop.nimble(21) execCmd
c:\binaries\nim-0.19.2\lib\system\nimscript.nim(237) exec
c:\binaries\nim-0.19.2\lib\system\nimscript.nim(237, 7) Error: unhandled exception: FAILED: nim c -r tests/tmath.nim
Command exited with code 1
7z a -r buildlogs-win-pkgs.zip %USERPROFILE%\.nimble\pkgs
7-Zip 18.05 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2018-04-30
```